Finish Blue Ocean Docs

bitwiseman opened the pull request:
WIP - not ready for review yet.

MarkEWaite commented on content/doc/book/blueocean/activity.adoc:
```diff
@@ -18,32 +18,69 @@ repository Pipeline.
 == Activity
 
+The default subsection of Activity Viewer shows a list of the latest
+completed or in-progress Runs of a Pipeline.
```
Intentionally capitalizing "Runs"?

bitwiseman replied:
Yeah, I'm thinking they are a specific thing/concept - a Pipeline Run.  
But I could be wrong.

rtyler replied:
I think capitalizing "Run" is fine, but the pluralization plus "of a Pipeline" makes this sentence awkward to me.

I think "in-progress Runs." would be fine, but the use of it as "of a Pipeline" makes for too many Capitalized Noun Things in the Sentence. :smiling_imp:

bitwiseman replied:
@rtyler - Okay, well, it is easy enough to fix over capitalization in the final edit.

MarkEWaite commented on content/doc/book/blueocean/activity.adoc:
```diff
@@ -18,32 +18,69 @@ repository Pipeline.
+Runs that have completed can be re-run by clicking the
+"Re-run" symbol (a counter-clockwise arrow "&#8634;").
+The list can be filtered by branch or pull request by clicking on the "branch"
+drop-down in the list's header.
```
Would it be as clear to say "list header" as "list's header"?  Possessive use for the list seems like it might not be required in this case.

MarkEWaite commented on content/doc/book/blueocean/activity.adoc:
```diff
@@ -18,32 +18,69 @@ repository Pipeline.
+The list can be filtered by branch or pull request by clicking on the "branch"
+drop-down in the list's header.
 
+This list does not allow runs to be edited or marked as favorites, those actions can be done from the
```
Break the sentence at the comma? 

> or marked as favorites, those actions can be done

becomes

> or marked as favorites. Those actions can be done

MarkEWaite commented on content/doc/book/blueocean/activity.adoc:
```diff
@@ -18,32 +18,69 @@ repository Pipeline.
+<<pipeline-editor#, pipeline editor>> on the Pipeline for that brach.
+Clicking the "Favorite" symbol (a star outline "&#9734;") adds a branch to the favorites list shown on the
+<<dashboard#favorites, Dashboard's "Favorites" list>> for this user.
+A favorite branch will show a solid star "&#9733;" and clicking it start will remove
```
I think instead of

> clicking it start will remove

you want

> clicking the solid star will remove

MarkEWaite commented on content/doc/book/blueocean/activity.adoc:
```diff
@@ -18,32 +18,69 @@ repository Pipeline.
 
+The "Branches" subsection shows a list of all branches that have a
+completed or in-progress Run in the current Pipeline.
+Each line in the list corresponds to a branch in SCM, showing
```
Should the first use of the acronym "SCM" use the expansion of the term followed by the acronym (as in, "software configuration management system (SCM))?

bitwiseman commented:
<img width="601" alt="screen shot 2017-06-02 at 5 32 09 pm" src="https://cloud.githubusercontent.com/assets/1958953/26749075/8a2b464c-47b9-11e7-9651-cdd5b8fd8430.png">
<img width="644" alt="screen shot 2017-06-02 at 5 32 23 pm" src="https://cloud.githubusercontent.com/assets/1958953/26749078/91031b52-47b9-11e7-90d2-2316d3f58046.png">

bitwiseman commented:
<img width="650" alt="screen shot 2017-06-02 at 5 34 43 pm" src="https://cloud.githubusercontent.com/assets/1958953/26749096/cea6d570-47b9-11e7-838a-9732b1b6ef89.png">

MarkEWaite commented on content/doc/book/blueocean/dashboard.adoc:
```diff
@@ -21,27 +21,77 @@ image:blueocean/dashboard/overview.png[Overview of the Dashboard, role=center]
+Blue Ocean represents the overall health of a Pipeline or Pipeline branch using
+weather icons, which change depending on the number of recent builds that have Passed.
+Health icons on the dashboard represent overall Pipeline health.
+Health icons in the Activity Branches view represent health for each branch.
```
Is "Activity Branches" enough of an independent concept or UI element that it should be quoted, similar to how you've quoted "Pull Requests" and "Merge Requests"?

bitwiseman commented:
@MarkEWaite 
Re red-ovals: I did something like that for the run details:
https://github.com/jenkins-infra/jenkins.io/blob/master/content/doc/book/resources/blueocean/pipeline-run-details/overview.png

I could do something similar, but frankly I'm trying to just get this shipped.  Unless you feel strongly about it, I'd rather let that go by in this case.

MarkEWaite commented:
No problem that the screenshots don't get the extra edits.

i386 commented on content/doc/book/blueocean/pipeline-editor.adoc:
```diff
@@ -47,16 +47,94 @@ The editor can be launched via:
+The "Environment" sections lets us set environment variables for the Pipeline.
+This is the same as the <<../pipeline/syntax#environment, "environment" directive>>.
+
+== Stageflow editor
```
Change all references s/Stageflow editor/Stage editor/

i386 commented on content/doc/book/blueocean/pipeline-editor.adoc:
```diff
@@ -47,16 +47,94 @@ The editor can be launched via:
+image:blueocean/editor/stageflow-basic.png[Simple Stageflow, role=center]
+
+Stages can be added to the Pipeline by clicking the "+" node to the right of an existing stage.
+Parallel branches can be added to a stage by clickingthe "+" node below an existing Stage.
```
Use s/parallel branches/Parallel stages/ instead. This becomes more correct terminology when we merge in the work to make this truely parallel stages so people need to get used to it.

i386 commented on content/doc/book/blueocean/pipeline-editor.adoc:
```diff
@@ -47,16 +47,94 @@ The editor can be launched via:
+image:blueocean/editor/stageflow-error.png[Stageflow with error, role=center]
+
+
+== Stages details
```
replace 's/Stages details/Stage configuration/

i386 commented on content/doc/book/blueocean/pipeline-editor.adoc:
```diff
@@ -47,16 +47,94 @@ The editor can be launched via:
+
+image:blueocean/editor/stage-details.png[Stage Details, role=center]
+
+The name of the Stage can be set at the top of the Stage details pane.
```
Use 'sheet' instead of 'pane'

i386 commented on content/doc/book/blueocean/getting-started.adoc:
```diff
@@ -79,8 +80,23 @@ If this is a new Jenkins instance, Blue Ocean will display a box offering to
+* *Jenkins* - Navigate to the Dashboard (reload if already viewing it)
+* *Pipelines* - Navigate to the Dashboard (do nothing if already viewing it)
+* *Administration* - <<../managing#, Manage>> this Jenkins instance (using the Classic UI)
+* *Switch to "Classic" UI* (arrow and square icon) - <<#switch-to-classic, Switch to the "Classic">>  Jenkins UI
```
why not place the icon here instead of describing it? You can get a PNG or SVG at https://material.io/icons/#ic_exit_to_app

bitwiseman replied:
@i386 - it is shown below in the linked section.  `blueocean/intro/switch-classic.png` .

i386 commented on content/doc/book/blueocean/pipeline-editor.adoc:
```diff
@@ -47,16 +47,94 @@ The editor can be launched via:
+== Pipeline settings
+
+By default, the right side of editor shows the "Pipeline Settings".
+This pane can be accessed by clicking anywhere in the
```
Use 'sheet' instead of 'pane'

bitwiseman replied:
I'll do this everywhere.

i386 commented on content/doc/book/blueocean/pipeline-editor.adoc:
```diff
@@ -47,16 +47,94 @@ The editor can be launched via:
+By default, the right side of editor shows the "Pipeline Settings".
+This pane can be accessed by clicking anywhere in the
+<<#stage-flow, Stageflow editor>> pane
+that is not a Stage or one of the "Add Stage" nodes.
```
`that is not a Stage or one of the "Add stage" buttons`

i386 commented on content/doc/book/blueocean/pipeline-editor.adoc:
```diff
@@ -47,16 +47,94 @@ The editor can be launched via:
+
+image:blueocean/editor/stageflow-basic.png[Simple Stageflow, role=center]
+
+Stages can be added to the Pipeline by clicking the "+" node to the right of an existing stage.
```
Button not Node

i386 commented on content/doc/book/blueocean/pipeline-editor.adoc:
```diff
@@ -47,16 +47,94 @@ The editor can be launched via:
+
+== Stages details
+
+Selecting a node in the Stageflow pane will open the "Stage Details" pane on the right side.
```
Selecting a stage

i386 commented on content/doc/book/blueocean/activity.adoc:
```diff
@@ -13,37 +13,89 @@ wip: true
 = Activity View
 
-The Blue Ocean Activity Viewer shows the all activity related to one
+The Blue Ocean Activity View shows the all activity related to one
```
I would use the the terms `Pipeline Activity tab`, `Pipeline Branches tab` and `Pipeline Pull Requests tab`

bitwiseman replied:
@i386 - I switched from "subsection" to "tab", but didn't add "Pipeline" to them.

i386 commented on content/doc/book/blueocean/activity.adoc:
```diff
@@ -13,37 +13,89 @@ wip: true
 
-The Blue Ocean Activity Viewer shows the all activity related to one
+The Blue Ocean Activity View shows the all activity related to one
 repository Pipeline.
```
"to one Pipeline"

bitwiseman commented:
Updated to address @i386's feedback: 
s/subsection/tab/  (extended to pages not previously in this PR)
s/pane/sheet/ 
s/Stageflow/Stage Editor/
s/Stage details/Stage configuration/
s/Step details/Step configuration/ (not mentioned by @i386, but seemed good for consistency)

While making these changes, noticed some missing run detail info and added that as well.

i386 commented:
@bitwiseman thanks so much 👍
